doc/user/content/manage/user-management/rbac-intro.md

```diff
@@ -0,0 +1,122 @@
+invite users to your Materialize organization, you have the option to give the
+user more elevated privileges in the Materialize administrator console.
+
+## RBAC structure
+
+RBAC in practice is a group of roles with assigned privileges and attributes.
+You can assign specific users to roles or assign privileges to users to inherit
+from other roles.
+
+### Roles
+
+A role is a collection of permissions you can apply to users. Roles make it
+easier to assign or revoke privileges on Materialize objects. You can group
+users into specified roles with different levels of permissions and adjust those
+permissions to ensure they have the correct level of access to objects.
+
+### Role attributes
+
+Role attributes are actions available to any role you create. Attributes are
+independent of any other object in Materialize and apply to the entire
+organization. You can edit these actions when you create the role:
+
+| Name            | Description                                                                     |
+|-----------------|---------------------------------------------------------------------------------|
+| `CREATEDB`      | Can create a database.                                                          |
+| `CREATEROLE`    | Can create, alter, drop, grant membership to, and revoke membership from roles. |
+| `INHERIT`       | Can inherit the privileges of roles that it is a member of. On by default.      |
+| `CREATECLUSTER` | Can create a cluster.                                                           |
+
+Currently, `INHERIT` cannot be turned off.
+
+### Privileges
+
+Privileges are the actions or operations a role is allowed to perform on a
+specific object. After you create a role, you can select specific privileges for that role based on the
+objects that role needs to access.
+
+Materialize supports the following privileges:
+
+| Privilege   | Description                                                  | `psql`   |
+| :---------: | :----------------------------------------------------------: | :------: |
+| `SELECT`    | Allows selecting rows from an object.                        | `r`      |
+| `INSERT`    | Allows inserting into an object.                             | `a`      |
+| `UPDATE`    | Allows updating an object (requires `SELECT`).               | `w`      |
+| `DELETE`    | Allows deleting from an object (requires `SELECT`).          | `d`      |
+| `CREATE`    | Allows creating a new object within another object.          | `U`      |
+| `USAGE`     | Allows using an object or looking up members of an object.   | `C`      |
+
+Note that `psql` returns an abbreviation of the privilege name.
+
+Objects in Materialize have different levels of privileges available to them.
+Materialize supports the following object type privileges:
+
+|     Object Type      |             Privileges              |
+|:--------------------:|:-----------------------------------:|
```

Comment:
nit: I think left-alignment is easy to scan and also more conventional in tech docs.